serverless/discover/about-serverless.adoc

```diff
@@ -26,12 +26,25 @@ You can propagate an event from an xref:../../serverless/discover/knative-event-
 
 The set of supported features, configurations, and integrations for {ServerlessProductName}, current and past versions, are available at the link:https://access.redhat.com/articles/4912821[Supported Configurations page].
 
-// only included for OCP currently until CRD docs are available in OSD
 ifdef::openshift-enterprise[]
+[id="about-serverless-scalability-performance"]
+== Scalability and performance
+
+{ServerlessProductName} has been tested with a configuration of 3 main nodes and 3 worker nodes, each of which has 64 CPUs, 457 GB of memory, and 394 GB of storage each.
+
+Using this configuration, it is possible to allocate 924 pods with a test application before the cluster fails to create new pods.
+
+The maximum number of Knative services that can be created using this configuration is 3,000. This corresponds to the xref:../../scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc#cluster-maximums-major-releases_object-limits[{product-title} Kubernetes services limit of 10,000], since 1 Knative service creates 3 Kubernetes services on average.
+
+The average scale from zero response time was 3.43 seconds, with a maximum response time of 8 seconds, and a 99.9th percentile of 4.576 seconds.
```

Comment:
I'm not sure that we should cite the exact numbers here without the full context (what exact application is used to create the pods and measure the runtime ? What was the network setup ?). Don't get me wrong, all of these are valuable information but valid for a specific setup that is not necessarily extrapolatable to other apps (it really also depends on the application itself how fast it starts, compare Quarkus (sub-second) to a JBoss EAP server (30s).

Reply:
Agree, we should use the word approximate and about. 
We should remove this as well : since 1 Knative service creates 3 Kubernetes services on average.

The average scale from zero response time was 3.43 seconds, with a maximum response time of 8 seconds, and a 99.9th percentile of 4.576 seconds. "for a simple quarkus application." Time might vary depending on the application and the runtime of the application.

Reply:
I wasn't totally clear on the ask here, updated to:

_The average scale from zero response time was 3.43 seconds, with a maximum response time of 8 seconds, and a 99.9th percentile of 4.576 seconds for a simple Quarkus application. These times might vary depending on the application and the runtime of the application._

Is this correct @nainaz @rhuss ?

Reply:
This looks good to me now.

Reply:
Looks good but maybe round 3.43 to 3.4 seconds and 4.576 second to 4.6 seconds in order to avoid a false impression of exactness, making it more like a general statement (i would maybe even round to 3.5s and 4.5s)